packages/ppx_test/ppx_test.1.7.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+version: "1.7.0"
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+homepage: "https://gitlab.com/camlspotter/ppx_test"
+bug-reports:
+  "https://gitlab.com/camlspotter/ppx_test/-/issues"
+dev-repo: "git+https://gitlab.com/camlspotter/ppx_test"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {build & >= "2.0"}
```

Comment:
Please remove the build tag https://github.com/ocaml/opam-repository/issues/14185
Also, it seems from your dune-project file that you only need >= 1.11